src/DateTime.js

```diff
@@ -465,7 +465,11 @@ export default class Datetime extends React.Component {
 		}
 
 		if ( (thisProps.value || thisProps.value === '') && thisProps.value !== prevProps.value ) {
-			this.setViewDate( thisProps.value );
+			if (thisProps.value) {
+				this.setViewDate( thisProps.value );
+			} else {
+				this.setViewDate( new Date());
```

Comment:
can we look into react-datetime source code and see what thay use to set the current date? I am just concern about the format in which the date is set.

Reply:
I had a look and they also use new Date(), therefore it should be fine.